e2e/rook-test-framework/clients/k8s_rook_block.go

```diff
@@ -0,0 +1,138 @@
+package clients
+
+import (
+	"errors"
+	"github.com/rook/rook/e2e/rook-test-framework/contracts"
+)
+
+type k8sRookBlock struct {
+	transportClient contracts.ITransportClient
+}
+
+var (
+	listCmd              = []string{"rook", "block", "ls"}
```

Comment:
Yeah we need to change this, we were initially under the impression lib was not available, we will refactor but not for this release